src/core/database/storage/ProfileModelStorage.ts

```diff
@@ -93,7 +92,7 @@ export class ProfileModelStorage extends VersionedObjectStorage<Record<string, P
                                 modified[name] = {
                                     ...modified[name],
                                     generationHash: defaultTestnetNetworkConfig.networkConfigurationDefaults.generationHash,
-                                    selectedNodeUrlToConnect: _.sample(defaultTestnetNetworkConfig.nodes).url, // Random url.
+                                    selectedNodeUrlToConnect: 'http://ngl-dual-001.testnet.symboldev.network:3000', // Random url.
```

Comment:
hardcode URL?

Reply:
it was picking some random url from the hardcoded ones, so to remove the ones in the Network config, had to add one here instead.

Reply:
No static nodes, please. Let's please utilize the statistics service.

Reply:
@0x6861746366574 I don't see it handy to pull statistics service here, we can leave it empty or add 'localhost:3000' for example.

Reply:
@bassemmagdy When we bring the nodes down, it will cause a crash. ;) 

No harm in adding the Statistics Service. It's a bit of (extra) work now, but it will save us work (and debt) in the future.

Reply:
As I understand this is extsing profiles migration from old testnet to new testnet which happens months ago (April). 
I think we can:
- keep as is. since this migration probably aleady executed on 99% of users (I assume that migration will not execute again) that had before old testnet profiles. And if it will execute to new ones it only causes that user will be switched to "dead" node. But he can switch in node selector.
- remove this migration. I don't know potential problems.

Implementing statistic services here is overkill IMO if it require big effort.